nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -388,25 +388,6 @@ After going through this checklist, you're ready to go!
 	else if(canopy_open)
 		add_overlay("canopy_open")
 
-/obj/structure/overmap/fighter/ai
-	ai_controlled = TRUE
-	ai_behaviour = AI_AGGRESSIVE
-	weapon_safety = FALSE
-	prebuilt = TRUE
-	faction = "nanotrasen"
-
-/obj/structure/overmap/fighter/ai/syndicate

```

Comment:
The based fighter nerf we've all been waiting for.

Reply:
They have been consolidated into overmap\types.dm so there is easy access to all NPC ships.
That file also should be renamed later.